examples/sample.conf

```diff
@@ -1,32 +1,65 @@
 # vim: ft=dosini
 
-site = ArchWiki
-
-debug = true
+[DEFAULT]
+#
+# Common options.
+#
 cache-dir = ~/.cache/wiki-scripts/
+; debug = true
 
-[ArchWiki]
+#
+# Database options.
+#
+db-dialect = postgresql
+db-driver = psycopg2
+db-user = wiki-scripts
+db-password = wiki-scripts
+db-host = localhost
+
+#
+# Archwiki-related options. Replace them with the needed Mediawiki API
+# if you use wiki-scripts on anothed Wiki project.
+#
+site = ArchWiki
```

Comment:
Now that the `--site` option was removed, the example should be updated.

Reply:
ws/client/connection.py uses `--site` for naming cookie file:
```python
cookie_file = args.cache_dir + "/" + args.site + ".cookie"
```
I have added `add_argument("--site")` to this file directly in `set_argparser()` method.
Besides, `site` is used for configuration file interpolation:
```
cookie-file = ${cache-dir}/${site}.cookie
```
Tell me if you wish to remove this option completely.